listenbrainz/webserver/templates/user/base.html

```diff
@@ -12,6 +12,8 @@
         <li {{ 'class=active' if active_section=='charts' }}><a href="{{ url_for('user.charts', user_name=user.musicbrainz_id) }}">Charts</a></li>
         {% if config.get("FEATURE_PLAYLIST", False) %}
         <li {{ 'class=active' if active_section=='playlists' }}><a href="{{ url_for('user.playlists', user_name=user.musicbrainz_id) }}">Playlists</a></li>
+        <li {{ 'class=active' if active_section=='recommendations' }}><a href="{{ url_for('user.recommendations', user_name=user.musicbrainz_id) }}">Recommendations</a></li>
+        <li {{ 'class=active' if active_section=='collaborations' }}><a href="{{ url_for('user.collaborations', user_name=user.musicbrainz_id) }}">Collaborations</a></li>
```

Comment:
are these playlist items? I think we should name the url functions `playlist_whatever` to make it clearer